tests/testthat/test-textstat_readability.R

```diff
@@ -1,4 +1,4 @@
-library("quanteda")
+library(quanteda)
```

Comment:
I always use quotes because I think package names should be literals rather than evaluated as objects. I agree with Yihui Xie and the JSS on this.


> One thing off-topic while I’m talking about these two functions: the argument character.only = FALSE for library() and require() is a design mistake in my eyes. It seems the original author(s) wanted to be lazy to avoid typing the quotes around the package name, so library(foo) works like library("foo"). Once you show people they can be lazy, you can never pull them back. Apparently, the editors of JSS (Journal of Statistical Software) have been trying to promote the form library("foo") and discourage library(foo), but I do not think it makes much sense now or it will change anything. If it were in the 90’s, I’d wholeheartedly support it. It is simply way too late now. 

https://yihui.org/en/2014/07/library-vs-require/

But...whatever.